docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,663 @@
+        },
+    });
+
+    console.log("results: ", results);
+```
+
+Based on the results returned by the nodes, we can execute the signed transactions by sending them to the blockchain networks.
+
+```js
+
+    if (results.signatures == undefined) {
+        return;
+    }
+
+    else if (results.signatures.chainBSignature == undefined) {
+        console.log("executing clawbackA tx..")
+        await executeTxA(results, chainAProvider);
+    }
+
+    else if (results.signatures.chainASignature == undefined) {
+        console.log("executing clawbackB tx..")
+        await executeTxB(results, chainBProvider);
+    }
+
+    else {
+        console.log("executing swap txs..")
+        await executeTxA(results, chainAProvider);
+        await executeTxB(results, chainBProvider);
+    }
+}
+```
+
+These two functions will handle sending individual transactions on each network.
+
+```js
+
+async function executeTxA(results, chainAProvider) {
+    const signatureA = formatSignature(results.signatures.chainASignature);
+    const tx1 = await chainAProvider.sendTransaction(
+        ethers.utils.serializeTransaction(
+            results.response.chainATransaction,
+            signatureA
+        )
+    );
+    console.log(tx1);
+    
+    const receipt1 = await tx1.wait();
+    const blockExplorer1 = LIT_CHAINS[chainAParams.chain].blockExplorerUrls[0];
+    
+    console.log(`tx: ${blockExplorer1}/tx/${receipt1.transactionHash}`);
+}
+
+async function executeTxB(results, chainBProvider) {
+    const signatureB = formatSignature(results.signatures.chainBSignature);
+    const tx2 = await chainBProvider.sendTransaction(
+        ethers.utils.serializeTransaction(
+            results.response.chainBTransaction,
+            signatureB
+        )
+    );
+    const receipt2 = await tx2.wait();
+    const blockExplorer2 = LIT_CHAINS[chainBParams.chain].blockExplorerUrls[0];
+
+    console.log(`tx: ${blockExplorer2}/tx/${receipt2.transactionHash}`);
+}
+```
+
+A signed transaction returned by the Lit Network has a different structure for signatures that we need to reconstruct as follows:
+
+```js
+function formatSignature(signature) {
+    const encodedSig = ethers.utils.joinSignature({
+        v: signature.recid,
+        r: `0x${signature.r}`,
+        s: `0x${signature.s}`,
+    });
+    return encodedSig;
+}
+```
+
+You can also explore other methods, such as checking permissions on the PKP, checking the fund status of each wallet, or the PKP itself [here](https://github.com/anshss/lit-evm-swap/blob/main/lit/utils.js).
+
+## Summary
+- Created transactions and clawback transactions objects based on swap parameters.
+- Wrote a Lit Action that returns signed transactions based on conditions.
+- Minted a PKP and assigned it the Lit Action.
+- Burned the PKP to restrict its signing ability to the Lit Action only.
+- Deposited funds to the PKP on both chains.
+- Executed the Lit Action on the Lit Network to receive signed transactions.
```

Comment:
"Executed the Lit Action on the Lit network to return our signed transaction objects"